docs/notebooks/asynchronous_greedy_multiprocessing.pct.py

```diff
@@ -83,7 +83,7 @@ def build_model(data):
 
 
 # these imports will be used later for optimization
-from trieste.acquisition import LocalPenalizationAcquisitionFunction
```

Comment:
did you try it out with Fantasizer? should work as well, right?

Reply:
I haven't tried. Indeed it should work. As suggested I'll add this to the integration tests (next PR).